specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5553 @@
+          "type": "string"
+        },
+        "requestMessage": {
+          "description": "Request message.",
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/PrivateEndpointServiceConnectionStatus",
+          "description": "Indicates whether the connection has been Approved/Rejected/Removed by the owner of the service."
+        }
+      },
+      "description": "Properties of a shared private link resource."
+    },
+    "EncryptionProperty": {
+      "properties": {
+        "status": {
+          "description": "Indicates whether or not the encryption is enabled for the workspace.",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "EncryptionStatus",
+            "modelAsString": true
+          }
+        },
+        "keyVaultProperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Customer Key vault properties."
+        }
+      },
+      "required": [
+        "status",
+        "keyVaultProperties"
+      ],
+      "type": "object"
+    },
+    "KeyVaultProperties": {
+      "properties": {
+        "keyVaultArmId": {
+          "description": "The ArmId of the keyVault where the customer owned encryption key is present.",
+          "type": "string"
+        },
+        "keyIdentifier": {
+          "description": "Key vault uri to access the encryption key.",
+          "type": "string"
+        },
+        "identityClientId": {
+          "description": "For future use - The client id of the identity which will be used to access key vault.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "keyIdentifier",
+        "keyVaultArmId"
+      ],
+      "type": "object"
+    },
+    "LinkedServiceResponse": {
+      "description": "Linked service.",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ResourceId of the link of the linked service.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Friendly name of the linked service.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of linked service.",
+          "readOnly": true
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedServiceProps"
+        }
+      }
+    },
+    "LinkedServiceRequest": {
+      "type": "object",
+      "description": "object used for creating linked service.",
+      "properties": {
+        "name": {
+          "description": "Friendly name of the linked service",
+          "type": "string"
+        },
+        "identity": {
```

Comment:
All resources supporting MSI need a location as well

Reply:
thank you, will add location.